templates/includes/company/certified_page_header.tx

```diff
@@ -2,7 +2,8 @@
         <div id="content" class="content-override"></div>
         <a href="/" class="govuk-back-link">Back</a>
         <h1 class="heading-xlarge" id="add-document-heading">Add documents to order</h1>
-        <p class="heading-large"><% $company.company_name %></p>
+        <p class="heading-large certified"><% $company.company_name %></p>
+        <p></p>
```

Comment:
Just wondering why we are adding empty paragraph tags after the ones we're populating with company name

Reply:
Removed now and updated css to provide the spacing.